paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/activity/PaymentSheetPlaygroundActivity.kt

```diff
@@ -401,6 +402,21 @@ class PaymentSheetPlaygroundActivity : AppCompatActivity() {
             customer = viewModel.customerConfig.value,
             googlePay = googlePayConfig,
             defaultBillingDetails = defaultBilling,
+            shippingDetails = if (viewModel.getSavedToggleState().setShippingAddress) {
```

Comment:
If the shipping address toggle is enabled, you can test this feature without going through the autocomplete flow.